commercetools-sdk-base/src/main/java/io/sphere/sdk/annotations/HasUpdateAction.java

```diff
@@ -37,5 +37,10 @@
     Class[] superInterfaces() default {};
     boolean makeAbstract() default false;
 
+    /**
+     * Specifies whether there should be a default factory method generated.
```

Comment:
Would be nice if we could add a short description of what a default factory method is. Is it a factory method which provides all update action properties as parameters?

Reply:
actually it provides all required update actions as parameter, this was done to have more control over the generation process.
But i agree i should add more javadoc there 👍